applications/DEMApplication/custom_utilities/inlet.cpp

```diff
@@ -601,6 +613,8 @@ namespace Kratos {
 
                 const double mass_that_should_have_been_inserted_so_far = mass_flow * (current_time - inlet_start_time);
 
+                std::uniform_int_distribution<> distrib(0, valid_elements_length - 1);
```

Comment:
Is it possible to create this object in the constructor? If so, it would be created only once. It should be possible to save it as a member variable.

Reply:
I think there is no problem in creating the object here since it is outside of a loop

Reply:
Me neither but to be sure that this will not slow down the simulation I will create it in the constructor as you said.

Reply:
After a while, I think it is necessary to create this object in that line. When you create this object you have to provide two arguments `distrib(int, int)` so you need to know the value of the `valid_element_length` before `distrib` is created.